generators/server/templates/src/main/java/package/service/mapper/_UserMapper.java

```diff
@@ -54,7 +53,7 @@ default User userFromId(<% if (databaseType == 'mongodb' || databaseType == 'cas
     }
 
     default Set<Authority> authoritiesFromStrings(Set<String> strings) {
-        return strings.stream().map(string -> {
+        return strings == null ? null : strings.stream().map(string -> {
```

Comment:
I'm sorry but I don't understand the use case. How do you save EntityA? Just doing a `.persist()`? Or you add it to an Hibernate-managed User?